src/oaep.rs

```diff
@@ -174,48 +222,117 @@ impl fmt::Debug for Oaep {
 ///
 /// [PKCS#1 OAEP]: https://datatracker.ietf.org/doc/html/rfc8017#section-7.1
 #[inline]
-pub fn encrypt<R: CryptoRngCore, K: PublicKey>(
+fn encrypt<R: CryptoRngCore + ?Sized, K: PublicKey>(
```

Comment:
For anyone else reviewing, this isn't a breaking change because the `oaep` module isn't currently public.

This PR makes it public though, and I approve of making these parts of the API surface private in that case.

Reply:
Let's split this to a separate commit to ease review.